### PR TITLE
Implement feedback for alpha 01

### PR DESCRIPTION
**Pull request by tggmbi**
Feedback included:

- AAR size was too large
- Lines were not smooth
- Needed properties for various XML attributes on InkView
- Support SDK level 23 and up

### CODE REVIEW

File: inksdk/ink/build.gradle [hunk 1]
```diff
@@ -32,7 +32,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
```
Comment (CesarValiente) [inline inksdk/ink/build.gradle:35]: In order to even decrease more the size of the binary maybe you want to add/enable some rules in the proguard files.

File: inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt [hunk 2]
```diff
@@ -166,24 +207,24 @@ class MainActivity : AppCompatActivity() {
...
-            var paint = Paint()
+            val paint = Paint()
 
             if (i > steps) i = 0 else i++
             paint.color = Color.argb(
                 255,
-                (Math.sin(frequency * i + 0) * 127 + 128).toInt(),
-                (Math.sin(frequency * i + 2) * 127 + 128).toInt(),
-                (Math.sin(frequency * i + 4) * 127 + 128).toInt()
+                (sin(frequency * i + 0) * 127 + 128).toInt(),
```
Comment (CesarValiente) [inline inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt:219]: Although this is for the sample code, I think is good if these values are also in constant variables with a meaningful name so others can understand better what's going on there.

File: inksdk/ink/build.gradle (hunk 1 above)
Comment (tggmbi) [reply inksdk/ink/build.gradle:35]: I will add it to the next version

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Meir :-)

**Pull request merged**